functorch/csrc/BatchRulesScatterOps.cpp

```diff
@@ -220,6 +224,33 @@ namespace {
 
     return std::make_tuple(self_, indices_, values_);
   }
+
+  auto index_put_plumbing_helper(const Tensor &self,
+                                 const List<optional<Tensor>> &indices,
+                                 const Tensor &values, int64_t cur_level) {
```

Comment:
nit: "unpackSelfAndIndicesAndValuesAtCurrentLevel" is many more characters but definitely more descriptive. It took me a couple of seconds before I understood what was going on here because we have a lot of functions named similarly: index_put_plumbing_helper, index_put_batch_rule_helper